profiling/src/profiling/mod.rs

```diff
@@ -690,7 +678,7 @@ impl Profiler {
     ) -> SampleMessage {
         // Lay this out in the same order as SampleValues
         static SAMPLE_TYPES: &[ValueType; 5] = &[
-            ValueType::new("sample", "count"),
+            ValueType::new("wall-samples", "count"),
```

Comment:
Are these not `time-samples` ? As they apply to CPU.
Does this imply `backend` changes ?